config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  get 'user/index'
```

Comment:
We removed the controller, so this route doesn't exists anymore! Delete this line, we will add some routes in the future